intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -410,7 +410,8 @@ \subsection{Generic resource requirements}
 \end{prompt}
 
 The job requires just one node, but it should have an Intel Westmere processor.
-A list with site-specific properties can be found in the next section.
+A list with site-specific properties can be found in the next section or in the 
+User Portal (``Available hardware''-section) of the VSC website.
```

Comment:
Can you put in a URL in a footnote, so they can simply click when reading the PDF?